.github/workflows/newbuild-vcpkg.yml

```diff
@@ -0,0 +1,333 @@
+            -password "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: "Remove builtin vcpkg"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: .
+        shell: "bash"
+        run: |
+          ${{ matrix.sudo }} rm -rf "$VCPKG_INSTALLATION_ROOT"
+
+      - name: Check out source code
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: "vcpkg Bootstrap"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: .
+        shell: "bash"
+        run: |
+          ./vcpkg/bootstrap-vcpkg.sh
+
+      - name: "Generate HPCC-Platform Build Files"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: .
+        shell: "bash"
+        run: |
+          mkdir build
+          cd build
+          cmake .. -DSKIP_ECLWATCH=ON -DCONTAINERIZED=OFF -DUSE_OPTIONAL=OFF ${{ matrix.cmake_config_options }}
+
+      - name: "Remove vcpkg build files"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: ./vcpkg
+        shell: "bash"
+        run: |
+          rm -rf ./buildtrees
+          rm -rf ./downloads
+
+      - name: "Build HPCC-Platform"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: ./build
+        shell: "bash"
+        run: |
+          cmake --build . ${{ matrix.cmake_build_options }}
+
+      - name: "Build HPCC-Platform"
+        if: ${{ contains(matrix.event_name, github.event_name) && needs.check-skip.outputs.platform }}
+        working-directory: ./build
+        shell: "bash"
+        run: |
+          cmake --build . ${{ matrix.cmake_build_options }}
+
+      - name: Bundle Package  
+        if: github.event_name == 'push' && github.ref == 'refs/tags/*'
+        shell: "bash"
+        run: |
+          cmake --build . --target package ${{ matrix.cmake_build_options }}
+
+      - name: Attach build artifacts
+        if: github.event_name == 'push' && github.ref == 'refs/tags/*'
+        uses: actions/upload-artifact@v3
+        with:
+          path: |
+            /hpcc-dev/HPCC-Platform/build/*.deb
+            /hpcc-dev/HPCC-Platform/build/*.rpm
+            /hpcc-dev/HPCC-Platform/build/*.exe
+            /hpcc-dev/HPCC-Platform/build/*.msi
+            /hpcc-dev/HPCC-Platform/build/*.dmg
+            /hpcc-dev/HPCC-Platform/build/*.pkg
+            
+  build-docker:
+    strategy:
+      matrix:
+        os:
+          [
+            "ubuntu-22.04",
+            "ubuntu-20.04",
+            "ubuntu-18.04",
+            "centos-8",
+            "centos-7",
+          ]
+        include:
```

Comment:
minor: see prev. comment, I don't think you need to list the variations in a matrix var. , if you are explicitly defining them via include's